packages/core/src/storybook/components/RelatedComponents.tsx

```diff
@@ -0,0 +1,48 @@
+import { linkTo as linkToStory } from '@storybook/addon-links'
+import React, { Children } from 'react'
+import { Box, Text, theme } from '../..'
+
+export type RelatedComponentProps = {
+  name: string
+  desc?: string
+  linkTo?: string
+  children: React.ReactNode
+}
+
+export const RelatedComponent = ({ name, desc, children, linkTo }: RelatedComponentProps) => (
+  <Box onClick={linkToStory(linkTo ?? name)} style={{ cursor: 'pointer' }}>
+    <Box
+      bg='background.light'
+      borderRadius='xl'
+      p={3}
+      style={{
+        display: 'grid',
+        placeItems: 'center',
+        aspectRatio: '2.5',
+      }}
+    >
+      {children}
+    </Box>
+    <Text textStyle='heading5' py={2}>
+      {name}
+    </Text>
+    {desc && <Text>{desc}</Text>}
+  </Box>
+)
+
+export type RelatedComponentContainerProps = {
+  children: React.ReactNode
+}
+
+export const RelatedComponentContainer = ({ children }: RelatedComponentContainerProps) => (
+  <Box
+    style={{
+      display: 'grid',
+      gridAutoFlow: 'column',
+      gridTemplateColumns: `repeat(${Children.count(children)}, 1fr)`,
+      gap: theme.space[3],
+    }}
+  >
```

Comment:
Would the `Container` component work for this?

Reply:
I looked at `Container`, and it seems to be a Box with a `size` prop to control width to some preset values. I don't think it accomplishes the same desired "grid-like" outcome,  

I have a [ticket](https://priceline.atlassian.net/jira/software/c/projects/UXPT/boards/4263?modal=detail&selectedIssue=UXPT-4286) on the JIRA board to introduce a `<Grid></Grid>` component to the design system, which I think would be perfect, and I intended to replace this implementations with a `Grid` as soon as it's available. 

Here's some [examples](https://chakra-ui.com/docs/components/grid/usage) of what we could do with the grid component! The CSS spec for grid is amazing, and since we don't have to worry as much about IE anymore we could lean in fully and make all sorts of [neat layouts](https://gridbyexample.com/examples/)